api/managementpb/backup/backups.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package backup.v1beta1;
+
+option go_package = "api/managementpb/backup;backupv1beta1";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+
+// RetryMode specifies how backup should retry in case of failure.
+enum RetryMode {
+  RETRY_MODE_INVALID = 0;
+  AUTO = 1;
+  MANUAL = 2;
+}
+
+message PerformBackupRequest {
+  string service_id = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  string location_id = 2 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  repeated string database_names = 3;
```

Comment:
as far as I know currently we don't support backup for exact databases.

Reply:
It's in arch design.
```
databaseNames: string[], required: false
note: "false" means it should backup all schemas found in the service
```
Everything outside of service_id and location_id is out of scope for this ticket. 
But I would keep it in API, even though is doesn't do anything. Maybe we can comment it, that for now it is unused.

Reply:
Yes, let's [reserve](https://developers.google.com/protocol-buffers/docs/proto3#reserved) the id = 3 and add a comment. I don't want to confuse people with a field which is not used actually.

Reply:
I commented it out. Is that reserved keyword good idea? When I read docs, it's said, that it should be used for deleted fields.